symplyphysics/laws/kinematic/distance_from_constant_acceleration.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## Motion with constant acceleration is described by the equation: S = S0 + V0 * t + (a * t^2)/2
```

Comment:
We already have this: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/kinematic/constant_acceleration_movement_is_parabolic.py